react/components/SKUSelector/utils/index.ts

```diff
@@ -51,7 +51,9 @@ export const isColor = (variation: string) => {
   if (!variation) return false
 
   return (
-    variation.toLowerCase() === 'cor' || variation.toLowerCase() === 'color' || variation.toLowerCase() === 'colour'
+    variation.toLowerCase() === 'cor' ||
+    variation.toLowerCase() === 'color' ||
+    variation.toLowerCase() === 'colour'
```

Comment:
I'm not sure how that information comes from the API, but could `variation` vary with the locale? For example, `"couleur"` for a store with French sales policy?